codegen/projections/high_score_service/sig/high_score_service/endpoint.rbs

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# WARNING ABOUT GENERATED CODE
+#
+# This file was code generated using smithy-ruby.
+# https://github.com/awslabs/smithy-ruby
+#
+# WARNING ABOUT GENERATED CODE
+
+module HighScoreService
+  module Endpoint
+    class Params < ::Struct[untyped]
+      attr_accessor endpoint: ::String
+    end
+
+    class Provider
+      def resolve_endpoint: (Params params) -> Hearth::RulesEngine::Endpoint
```

Comment:
I'm a little uncertain about making this Hearth generic since its a definition thats specific to the Rules Engine.   Esp the `AuthScheme` part - these are specific to RulesEngine view of AuthSchemes rather than our generic SRA defined auth schemes....